contributed_definitions/nyaml/NXcs_gpu_obj.yaml

```diff
@@ -5,7 +5,7 @@ symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
 type: group
-NXcs_gpu_obj(NXobject):  # NXcircuit_board ?
+NXcs_gpu_obj(NXcircuit):
```

Comment:
same here

Reply:
It can also be removed if it does not use any app def. Even if is used in appdef the appdef can modified. It also contains only name and fabrication.

Reply:
Agree, followed your suggestion and refactored this set of base classes, only NXcs_computer remains using NXcircuit.